src/web/src/main/java/tupperdate/web/routing/recipes/Recipes.get.kt

```diff
@@ -13,19 +13,38 @@ import tupperdate.web.model.Recipe
 import tupperdate.web.model.toRecipeDTO
 import tupperdate.web.util.await
 
+fun Route.recipesGet(store: Firestore) {
+    some(store)
+    all(store)
+}
+
+/**
+ * Retrieves a single recipe from its identifier.
+ *
+ * @param store the [Firestore] instance that is used.
+ */
+private fun Route.some(store: Firestore) = get("/{identifier}") {
+    val id = call.parameters["identifier"] ?: statusException(HttpStatusCode.BadRequest)
+
+    val document = store.collection("recipes").document(id).get().await()
+    val recipe = document.toObject(Recipe::class.java) ?: statusException(InternalServerError)
```

Comment:
We probaby want to return a NotFound error instead of an InternalServerError, which would be the most likely cause for an error at this point.

Reply:
Should be fixed in 12c9c79ddc65a6a5731e7031f19c2c2205dee0b8.